.changeset/unlucky-doors-cheer.md

```diff
@@ -0,0 +1,9 @@
+---
+"@patternfly/pfe-core": minor
+---
+
+✨ Added `ScrollSpyController`
+
+`ScrollSpyController` sets an attribute (`active` by default) on one of it's
+children when that child's `href` attribute is to a hash reference to an IDd
+heading on the page.
```

Comment:
Need a changeset entry here for the addition of `roving-tabindex-controller` as well

```Suggestion
✨ Added `ScrollSpyController`
✨ Added `RovingTabindexController`

- `ScrollSpyController` sets an attribute (`active` by default) on one of it's
children when that child's `href` attribute is to a hash reference to an IDd
heading on the page.
- `RovingTabindexController` implements roving tabindex, as described in WAI-ARIA practices.  
```